spec/automation/unit/method_validation/requested_spec.rb

```diff
@@ -253,7 +253,7 @@ def check_results(requested_hash, storage, cpu, vms, memory)
     it "remove a disk thats not found" do
       setup_model("vmware_reconfigure")
       @reconfigure_request.update_attributes(:options => {:src_ids => [@vm_vmware.id], :request_type => :vm_reconfigure,\
-      :disk_remove => [{"disk_name" => "not found", "persistent" => true, "thin_provisioned" => false,\
+      :disk_remove => [{:disk_name => "not found", "persistent" => true, "thin_provisioned" => false,\
```

Comment:
@billfitzgerald0120 Should all keys here be symbols e.g persistent, thin_provisioned?